### DISCUSSION TITLE
Just installed Orca but cannot setup a device as an instruction loop keeps running

### DISCUSSION THREAD

**Discussion by AndyDragon666** [OrcaSlicer/OrcaSlicer#2800] [Q&A]
Hi, For some strange reason as soon as I click on the Device tab an instruction loop starts running that I cannot stop to actually do what it is telling me to do! The top of the page asks me to set up my printer connection to view the device, except the only thing below it is the instructional loop.  

Recommend that that the text is changed to direct people to go the the prepare tab and add the connection to their printer.

**Comment by esue1943** [comment]
I'm having the same problem
